src/modules/color/index.ts

```diff
@@ -297,7 +297,9 @@ export class Color {
     let color: string | number[];
     let cssFunction: CSSFunction = 'rgb';
     if (format === 'hex') {
-      color = this.faker.datatype.hexadecimal({ length: includeAlpha ? 8 : 6 });
+      color = this.faker.datatype
+        .hexadecimal({ length: includeAlpha ? 8 : 6 })
+        .slice(2);
```

Comment:
Use the explicitly an empty prefix? If we have the feature, use it.
```suggestion
        .hexadecimal({ length: includeAlpha ? 8 : 6, prefix: '' })
```